dle/search/templates/search/elastic/search.html

```diff
@@ -34,110 +34,6 @@ <h3>Drug Label Generic Name</h3>
-            }, {
-                field: "drug_label_generic_name.keyword",
-                type: "string",
-                attribute: "drug_label_generic_name",
-            }
-        ],
-      }
-    })
-
-    const search = instantsearch({
-      indexName: "productsection",
-      searchClient: SearchkitInstantsearchClient(sk)
-    });
-
-    search.addWidgets([
-        instantsearch.widgets.searchBox({
-            container: "#searchbox"
-        }),
-        instantsearch.widgets.currentRefinements({
-            container: "#current-refinements"
-        }),
-        instantsearch.widgets.menuSelect({
-            container: "#section-name-filter",
-            attribute: "section_name",
-            limit: 100
-        }),
-        instantsearch.widgets.refinementList({
-            container: "#drug-label-source-filter",
-            attribute: "drug_label_source"
-        }),
-        instantsearch.widgets.menuSelect({
-            container: "#drug-label-product-name-filter",
-            attribute: "drug_label_product_name",
-            limit: 100
-        }),
-        instantsearch.widgets.menuSelect({
-            container: "#drug-label-generic-name-filter",
-            attribute: "drug_label_generic_name",
-            limit: 100
-        }),
-        instantsearch.widgets.hits({
-            container: "#hits",
-            templates: {
-                item(hit, { html, components }) {
-                    return html`
-                    <h2>
-                        ${components.Highlight({ attribute: 'drug_label_product_name', hit })}
-                    </h2>
-                    <h3>
-                        Generic Name: ${components.Highlight({ attribute: 'drug_label_generic_name', hit })}
-                    </h3>
-                    <h3>
-                        Section: ${components.Highlight({ attribute: 'section_name', hit })}
-                    </h3>
-                    <ul>
-                        <li>Source: ${hit.drug_label_source}</li>
-                        <li>Version Date: ${hit.drug_label_version_date}</li>
-                        <li>Product Number: ${hit.drug_label_product_number}</li>
-                        <li>Link: <a href="${hit.drug_label_link}">${hit.drug_label_link}</a></li>
-                    </ul>
-                    <p>${components.Snippet({ attribute: 'section_text', hit })}</p>
-                    `;
-                }
-            }
-        }),
-        instantsearch.widgets.pagination({
-            container: "#pagination"
-        }),
-        instantsearch.widgets.hitsPerPage({
-            container: '#hits-per-page',
-            items: [
-                { label: '10 hits per page', value: 10, default: true },
-                { label: '20 hits per page', value: 20 },
-                { label: '50 hits per page', value: 50 }
-            ],
-        })
-    ]);
-
-    search.start();
-  </script>
+{% load static %}
```

Comment:
Yeah, Django can pull static files from a bunch of different directories depending on how you configure it. That way you can have static files in different apps since you may distribute or reuse apps. Django copies them all into a new `static` folder in the application when you run the `collectstatic` management command. This happens during our `entrypoint` script. But to answer the question, yes whenever you're using a local static file, you need to use the `{% load static %}` tag as it will build the URL for the file based on your config. Not needed for external files.

Reply:
Feel free to ping on things like that, I do a lot of Django and sometimes there are gotchas like that which I might be able to identify quickly. Sometimes if you don't know what to Google they can take a long time to hunt down